config/routes.rb

```diff
@@ -1,5 +1,8 @@
 JobConstructor::Application.routes.draw do
-  resources :templates
```

Comment:
Better link:

http://guides.rubyonrails.org/v4.2/routing.html#restricting-the-routes-created